tests/fixtures/api_fixtures.py

```diff
@@ -20,6 +20,16 @@ def _api_create_or_update_host(
     return _api_create_or_update_host
 
 
+@pytest.fixture(scope="function")
+def api_post(flask_client):
+    def _api_post(
+        url, host_data, query_parameters=None, extra_headers=None, auth_type="account_number", identity_type="User"
+    ):
+        return do_request(flask_client.post, url, host_data, query_parameters, extra_headers, auth_type, identity_type)
```

Comment:
This is the only request fixture that passes in the _auth_type_ parameter. It passes in a default value though, thus it is redundant. Removing the parameter would make this function nicely aligned with its sister ones: _api_put_ and _api_put_. They are also very repetitive, they may be worth DRYING.

```suggestion
    def _api_post(url, host_data, query_parameters=None, extra_headers=None, identity_type="User"):
        return do_request(flask_client.post, url, host_data, query_parameters, extra_headers, identity_type)
```